src/main/java/com/ays/auth/model/enums/AysTokenClaims.java

```diff
@@ -18,8 +18,6 @@ public enum AysTokenClaims {
     USER_ID("userId"),
     USERNAME("username"),
     USER_TYPE("userType"),
-    USER_FIRST_NAME("userFirstName"),
-    USER_LAST_NAME("userLastName"),
```

Comment:
Bu iki alanı neden buradan kaldırdık?

Reply:
Token bilgisinde Admin için bu bilgilere gerek var mı? Sadece id kullanıyoruz.

Reply:
@Rapter1990 
> Token bilgisinde Admin için bu bilgilere gerek var mı? Sadece id kullanıyoruz.

Token içerisindeki bilgiler sadece BE tarafından kullanılmıyor, aynı zamanda FE tarafında da kullanılıyor

Reply:
Durum User içinde USER_FIRST_NAME ve USER_LAST_NAME kaldıramayız. Kaldıracaksak FE AuthUser'da bunları optional olarak tanımlanması lazım. Değer gelebilirde gelmeye bilirde şeklinde.

Reply:
> Durum User içinde USER_FIRST_NAME ve USER_LAST_NAME kaldıramayız. Kaldıracaksak FE AuthUser'da bunları optional olarak tanımlanması lazım. Değer gelebilirde gelmeye bilirde şeklinde.

Mobil tarafta bu bilgilere ihtiyacımız olmadığı için kaldırmayı düşündük, Kurum tarafında kullanıyoruz.

Reply:
@mehmetseymusyuzen yazılan yorumları, yorum sahibi kontrol edip resolve ediyor, sonrasında approve'luyor